oarepo_vocabularies/cli.py

```diff
@@ -1,7 +1,201 @@
+    records_not_added_by_ror = get_records_not_added_by_ror()
+
+    # convert ROR dump to YAML, and filter out records that are not added by ROR
+    tmp_ror_converted_dir = Path("/tmp/ror-converted")
+    if tmp_ror_converted_dir.exists():
+        click.secho(f"Removing {tmp_ror_converted_dir}", fg="yellow")
+        shutil.rmtree(tmp_ror_converted_dir)
+
+    tmp_ror_converted_dir.mkdir(parents=True)
+    convert_ror(
+        tmp_ror_file, tmp_ror_converted_dir / "ror.yaml", records_not_added_by_ror
+    )
+
+    # create a catalogue.yaml file to load the converted ROR dump
+    (tmp_ror_converted_dir / "catalogue.yaml").write_text(
+        """
+affiliations:
+- writer: affiliations
+  update: true
+- source: ./ror.yaml
+"""
+    )
+
+    # and load it
+    ctx.invoke(load, fixture_dir_or_catalogue=str(tmp_ror_converted_dir), verbose=True)
+
+
+def convert_ror(
+    tmp_ror_file: Path, output: Path, records_not_added_by_ror: dict[str, str]
+):
+
+    with open(output, "w", encoding="utf-8") as out_f:
+        yaml.safe_dump_all(
+            get_affiliation_records(tmp_ror_file, records_not_added_by_ror),
+            out_f,
+            default_flow_style=False,
+            allow_unicode=True,
+        )
+
+
+def get_affiliation_records(
+    tmp_ror_file: Path, records_not_added_by_ror: dict[str, str]
+) -> typing.Generator[dict[str, typing.Any], None, None]:
+    # unzip the file to get the csv file and open it
+    with zipfile.ZipFile(tmp_ror_file, "r") as zip_ref:
+        zf = [x for x in zip_ref.namelist() if x.endswith("schema_v2.csv")]
+        if not zf:
+            raise click.ClickException(
+                f"Failed to find schema_v2.csv in {tmp_ror_file}"
+            )
+        click.secho(f"Processing {zf[0]}", fg="yellow")
+        with zip_ref.open(zf[0], "r") as f:
+            reader = csv.DictReader(
+                io.TextIOWrapper(f, encoding="utf-8"),
+                delimiter=",",
+                quotechar='"',
+            )
+
+            for row in tqdm.tqdm(reader, leave=False):
+                names = ror_to_multidict(row["names.types.label"])
+                aliases = ror_to_multidict(row["names.types.alias"])
+                term_id = "ror:" + row["id"].rsplit("/", 1)[-1]
+                if row["id"] in records_not_added_by_ror:
+                    # if there is a database record with ROR and at the same time
+                    # it was not created from ROR, we need to skip it and not update
+                    # the name
+                    continue
+
+                if aliases:
+                    names = aliases
+
+                if not names:
+                    continue
+
+                if "no_lang_code" in names:
+                    n = names.pop("no_lang_code")
+                    if "en" not in names:
+                        names["en"] = n
+
+                name = names.get("cs", names.get("en")) or next(iter(names.values()))
+                yield {
```

Comment:
Can't we reuse logic from: https://github.com/oarepo/oarepo-vocabularies/blob/main/oarepo_vocabularies/authorities/providers/ror_provider.py#L119-L152?

Reply:
can't reuse it - the ror_display is primarily English and we'd like to have czech name if possible.